mlrun/model_monitoring/stream_processing.py

```diff
@@ -0,0 +1,756 @@
+import json
```

Comment:
A bit hard for me to review this whole file as I don't really know the requirements, suggest to use @yaronha here